docs/cloud/auditing.md

```diff
@@ -19,7 +19,7 @@ lakeFS Cloud
 
 The lakeFS audit log allows you to view all relevant user action information in a clear and organized table, including when the action was performed, by whom, and what it was they did. 
 
-![audit log](/assets/img/audit-log.png)
+![audit log](../assets/img/audit-log.png)
```

Comment:
Fixed some warnings for the IDE

Reply:
ok. I'd personally prefer to leave stuff like this (and the line wrapping) unchanged, since not everyone uses your IDE ;) and specifically on the paths it's easier for stuff to break as things move, whereas images are always going to be from the root. 

not blocking, just my personal preference :)

Reply:
> ok. I'd personally prefer to leave stuff like this (and the line wrapping) unchanged, since not everyone uses your IDE ;) and specifically on the paths it's easier for stuff to break as things move, whereas images are always going to be from the root.
> 
> not blocking, just my personal preference :)

@rmoff Thanks - I'll revert the changes for the image paths.
Regarding the line wrap - line break after 75 chars is a bit outdated...  (❤️ @arielshaqed )